text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/internal/parser/Token.java

```diff
@@ -126,6 +127,19 @@ public CharSequence get(final CharSequence message) {
     );
   }
 
+  @Override
+  public boolean equals(final Object other) {
```

Comment:
This is just used for tests, yes?

Reply:
yeah, so we can directly compare token output